include/ry/codegen.hpp

```
@@ -100,63 +101,16 @@ class CodeGen {
-
-private:
+    // Table-driven native call dispatch (public for use by self-registering
+    // stdlib packages in codegen_call_<pkg>.cpp files).
+    llvm::Value *emitTableDrivenNativeCall(const CallExpr &e,
+                                            const char *package,
+                                            const NativeDispatchEntry *table,
+                                            size_t table_size);
+
+    // Members below are accessed by free-function custom emitters in
+    // codegen_call_<pkg>.cpp files, so they must be public.
+public:
```

Clang (14.0.6)

[warning] redundant access specifier has the same accessibility as the previous access specifier

(readability-redundant-access-specifiers)